pkg/util/converter/prom.go

```diff
@@ -564,6 +564,10 @@ func readScalar(iter *jsonitere.Iterator) backend.DataResponse {
 		Custom: resultTypeToCustomMeta("scalar"),
 	}
 
+	if dataPlane {
+		frame.Meta.TypeVersion = data.FrameTypeVersion{0, 1}
+	}
```

Comment:
@kylebrandt could/should we unit test this? Seems we've added a conditional to our application code, but no test to specify the behaviour.